test/controllers/bigbluebutton_api_controller_test.rb

```diff
@@ -932,7 +932,7 @@ class BigBlueButtonApiControllerTest < ActionDispatch::IntegrationTest
     assert_response :success
     assert_select 'response>returncode', 'SUCCESS'
     assert_select 'response>messageKey', 'noRecordings'
-    assert_select 'response>message', 'There are no recordings for the meeting(s).'
+    assert_select 'response>message', 'There are no recordings for the meetings'
```

Comment:
This wording is intentional, to match the response produced by BigBlueButton. Please leave it unchanged.